src/components/Shared/VulnerabilityPackageCard.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Card, Stack, Typography } from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles';
+
+const useStyles = makeStyles(() => ({
+  cvePackageCard: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    background: '#FFFFFF',
+    boxShadow: '0rem 0.3125rem 0.625rem rgba(131, 131, 131, 0.08)',
+    border: '1px solid #E0E5EB',
+    borderRadius: '0.75rem',
+    flex: 'none',
+    alignSelf: 'stretch',
+    width: '100%'
+  },
+  cveInfo: {
+    marginTop: '2%'
+  }
+}));
+
+function VulnerabilityPackageCard(props) {
+  const { cve } = props;
+  let styles = useStyles();
+
+  return (
+    <Card className={styles.cvePackageCard}>
+      <Stack direction="column" spacing="0.4rem" display="flex" sx={{ padding: '0.5rem 0.5rem' }}>
+        <div>
```

Comment:
let's avoid empty divs in our JSX. A Grid would be preferable over stack in this specific example to achieve the template. But if you stick with Stack, replace the empty divs either with row stack or with react fragment

Reply:
Thanks for the suggestion! I've changed it to use Grid instead,